src/styles/blocks/card.scss

```
@@ -0,0 +1,114 @@
+@use '../utils/variables' as *;
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border: 1px solid $card-border-color;
+  border-radius: 10px;
+
+  background-color: $card-background-color;
+
+  &__info {
+    display: flex;
+    flex-direction: column;
+
+    box-sizing: border-box;
+    width: 100%;
+    padding: 0 16px;
+  }
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    object-fit: contain;
+    margin-top: 33px;
+  }
+
+  &__title {
+    margin-top: 40px;
+    margin-bottom: 4px;
+    font-size: 12px;
+    font-weight: $font-weight-medium;
+    line-height: 18px;
+    color: $title-color;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: $font-weight-regular;
+    line-height: 14px;
+    color: $code-color;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 16px;
+    gap: 17px;
+
+    > :last-child {
+      font-size: 10px;
+      font-weight: $font-weight-regular;
+      line-height: 14px;
+      color: $reviews-color;
+      margin: 0;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: baseline;
+    margin-top: 24px;
+
+    > :first-child {
+      font-size: 12px;
+      font-weight: $font-weight-regular;
+      line-height: 18px;
+      color: $price-color;
+      margin: 0;
+    }
+    > :last-child {
+      font-size: 16px;
+      font-weight: $font-weight-bold;
+      line-height: 18px;
+      color: $price-discount-color;
+      margin: 0;
+    }
+  }
+
+  &__buy {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    box-sizing: border-box;
+    width: 166px;
+    height: 40px;
+    border-radius: 5px;
+    margin: 16px auto;
+    padding: 12px;
+
+    font-size: 14px;
+    font-weight: $font-weight-bold;
+    line-height: 16px;
+    text-decoration: none;
+    text-align: center;
+    color: $button-color;
+    background-color: $button-background-color;
+    border: 1px solid transparent;
```

`box-sizing: border-box` style causes border to be included in the element’s width and height so using transparent border makes button smaller
```suggestion
    border: 1px solid $button-background-color;
```